dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java

```diff
@@ -72,6 +72,34 @@ public List<LambdaEvent> getLambdaEventsByOrganization(@ApiParam(hidden = true)
         return lambdaEventDAO.findByOrganization(organization, offset, limit, authorizedRepos);
     }
 
+    @GET
+    @Timed
+    @UnitOfWork(readOnly = true)
+    @Path("/{userid}/{organization}")
+    @Operation(operationId = "getUserLambdaEventsByOrganization", description = "Get all of the Lambda Events for the given user and GitHub organization.",
+            security = @SecurityRequirement(name = ResourceConstants.JWT_SECURITY_DEFINITION_NAME))
+    @ApiOperation(value = "See OpenApi for details")
+    public List<LambdaEvent> getUserLambdaEventsByOrganization(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
+           @ApiParam(value = "userid", required = true) @PathParam("userid") long userid,
+           @ApiParam(value = "organization", required = true) @PathParam("organization") String organization,
+           @ApiParam(value = PAGINATION_OFFSET_TEXT) @QueryParam("offset") @DefaultValue("0") String offset,
```

Comment:
Is this endpoint intended to be hooked up to the UI?  If yes, pagination is great.  If not, the endpoint could just return all of the events, and the admin/curator can use the tool of their choice to navigate them.

Reply:
Think the idea was back-end at least, ideally UI in the end

Reply:
Yes, the UI will be hooked up :)